### PR TITLE
Making the Getting Started docs more vertical

### PR DESCRIPTION
**Pull request by leoebfolsom**
The general structure we've tried to follow is:

- **Getting Started** for stuff that you _need_ to Get Started
- **Guides** for additional/optional stuff

The intent of the PR is to move us closer to that structure. Specifically, the user should not have to deviate from the Getting Started path to find the instructions for setting up Datafold in CI using one of the CI tools (GH Actions, GitLab, Citcle CI).

Along the way:

- Added screenshots
- Moved general info about CI from Guides > CI > Getting Started into Deployment Testing > Team Cloud > How It Works
- Copy edits

To do, but I think better to address in a separate PR:

- Review the GH Actions, GitLab, Citcle CI guides and consider how they could be improved
- Add an Airflow guide
- Additional copy editing
 
The intent of this PR is to make a big structural change that will have a positive impact today, not to get to a perfect state -- more PRs to follow!

Screenshot 2023-07-19 at 20 26 28

### CODE REVIEW

File: docs/deployment_testing/team_cloud/getting_started_for_customers/dbt/dbt_core.md [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
 pagination_prev: deployment_testing/team_cloud/getting_started_for_customers/source_control
 pagination_next: deployment_testing/team_cloud/getting_started_for_customers/data_apps
 ---
-## Configuration
+
+### Getting Started
+
+To add Datafold to your CI using dbt Core:
+
+#### 1. Create a dbt Core integration.
```
Comment (sungchun12) [inline docs/deployment_testing/team_cloud/getting_started_for_customers/dbt/dbt_core.md:14]: I recommend include images like this from the main account. The table is really overwhelming to digest and I won't read it all. This was true when I started in my own sandbox account. I don't have settings enabled on that account for whatever reason.

image

image

File: docs/deployment_testing/team_cloud/getting_started_for_customers/dbt/dbt_core.md [hunk 2]
```diff
@@ -23,14 +31,42 @@ Complete the configuration by specifying the following fields:
...
+
+#### 4. Add a Datafold command to your PR CI process to upload dbt artifacts (the `manifest.json`) representing the state of the project _in the latest commit of the PR branch_.
+
+That command will look something like this ...
+
+```
+datafold dbt upload --ci-config-id 123 --run-type pull_request --target-folder ./target/ --commit-sha 1e6f20d0b1337db5b6d2688c04e6d4ac
+```
+
+... but the details will depend on [which CI tool](#ci-implementation-tools) you use.
```
Comment (sungchun12) [inline docs/deployment_testing/team_cloud/getting_started_for_customers/dbt/dbt_core.md:48]: Hyperlink is broken

Comment (sungchun12) [reply docs/deployment_testing/team_cloud/getting_started_for_customers/dbt/dbt_core.md:48]: I recommend including a code snippet with a github action as the default. And write, "Example Github Action"

I'd also make a note to double their base branch is `main` or `master`

File: docs/deployment_testing/team_cloud/getting_started_for_customers/dbt/dbt_core.md [hunk 3]
```diff
@@ -23,14 +31,42 @@ Complete the configuration by specifying the following fields:
...
+
+#### 5. Add a Datafold command to your production CI process to upload dbt artifacts (the `manifest.json`) representing the state of the project _in the latest commit of the production branch_.
```
Comment (sungchun12) [inline docs/deployment_testing/team_cloud/getting_started_for_customers/dbt/dbt_core.md:50]: see above

File: docs/deployment_testing/team_cloud/getting_started_for_customers/dbt/dbt_core.md [hunk 4]
```diff
@@ -23,14 +31,42 @@ Complete the configuration by specifying the following fields:
...
+
+#### 3. Obtain an Datafold API Key.
+![](../../../../../static/img/api_key.png)
+
+#### 4. Add a Datafold command to your PR CI process to upload dbt artifacts (the `manifest.json`) representing the state of the project _in the latest commit of the PR branch_.
+
+That command will look something like this ...
+
+```
+datafold dbt upload --ci-config-id 123 --run-type pull_request --target-folder ./target/ --commit-sha 1e6f20d0b1337db5b6d2688c04e6d4ac
```
Comment (sungchun12) [inline docs/deployment_testing/team_cloud/getting_started_for_customers/dbt/dbt_core.md:45]:
```suggestion
datafold dbt upload --ci-config-id 123 --run-type ${DATAFOLD_RUN_TYPE} --commit-sha ${GIT_SHA}
```

Keep it consistent with the template yaml

File: docs/deployment_testing/team_cloud/getting_started_for_customers/dbt/dbt_core.md [hunk 5]
```diff
@@ -23,14 +31,42 @@ Complete the configuration by specifying the following fields:
...
+```
+
+... but the details will depend on [which CI tool](#ci-implementation-tools) you use.
+
+#### 5. Add a Datafold command to your production CI process to upload dbt artifacts (the `manifest.json`) representing the state of the project _in the latest commit of the production branch_.
+
+That command will look something like this ...
+
+```
+datafold dbt upload --ci-config-id 123 --run-type production --target-folder ./target/ --commit-sha 1e6f20d0b1337db5b6d2688c04e6d4ac
```
Comment (sungchun12) [inline docs/deployment_testing/team_cloud/getting_started_for_customers/dbt/dbt_core.md:55]:
```suggestion
datafold dbt upload --ci-config-id 123 --run-type ${DATAFOLD_RUN_TYPE} --commit-sha ${GIT_SHA}
```

Keep it consistent with the template yaml

File: docs/deployment_testing/team_cloud/how_datafold_team_cloud_works.md [hunk 6]
```diff
@@ -57,10 +73,7 @@ We'll walk through the setup steps in more detail in the [Getting Started](/depl
 - Two versions of your dbt project's `manifest.json` will be submitted to Datafold representing the state of production code as well as PR/MR code.
   - This submission of dbt artifacts happens out-of-the-box with dbt Cloud.
   - dbt Core users can set this up by adding steps to their existing CI configuration in Circle CI, GitHub Actions, or GitLab.
-    - [CircleCI](/guides/ci/circleci)
-    - [GitHub Actions](/guides/ci/github_actions.md)
-    - [GitLab](/guides/ci/gitlab_ci)
-- Datafold will use these two version of dbt the `manifest.json` to identify code differences.
-- Datafold will then query your warehouse and run Data Diffs of modified models and downstreams.
+- Datafold uses two versions of the `manifest.json` to identify code differences.
+- Datafold queries your warehouse and run Data Diffs of modified models and downstreams.
```
Comment (sungchun12) [inline docs/deployment_testing/team_cloud/how_datafold_team_cloud_works.md:77]:
```suggestion
- Datafold queries your warehouse and runs Data Diffs of modified models and other downstream impacts to data apps like Hightouch, Mode, etc.
```

File: docs/deployment_testing/team_cloud/how_datafold_team_cloud_works.md [hunk 7]
```diff
@@ -57,10 +73,7 @@ We'll walk through the setup steps in more detail in the [Getting Started](/depl
...
   - Got a huge dbt project with many downstreams? Don't worry--you can set up [Slim Diff](/guides/slim_diff.md) or utilize other [configuration options](/guides/dbt_advanced_configs.md) to manage scale while ensuring critical models are diffed.
 - The results of the Data Diffs are then written directly to GitHub/GitLab and can be viewed in the Datafold Cloud application.
```
Comment (sungchun12) [inline docs/deployment_testing/team_cloud/how_datafold_team_cloud_works.md:79]:
```suggestion
- The results of the Data Diffs are then written directly to GitHub/GitLab and more details can be viewed in the Datafold Cloud application.
```

File: docs/deployment_testing/team_cloud/getting_started_for_customers/dbt/dbt_core.md (hunk 2 above)
Comment (sungchun12) [reply docs/deployment_testing/team_cloud/getting_started_for_customers/dbt/dbt_core.md:48]: It's okay if there's duplicate writing, what matters is that the reader doesn't do too much context switching.

**Review by sungchun12 — CHANGES_REQUESTED** [review]
Made a couple edits, but overall like the direction.

**Comment by leoebfolsom** [comment]
Thank you @sungchun12 , follow-on commits forthcoming!

File: docs/deployment_testing/team_cloud/getting_started_for_customers/dbt/dbt_core.md (hunk 1 above)
Comment (sungchun12) [reply docs/deployment_testing/team_cloud/getting_started_for_customers/dbt/dbt_core.md:14]: We can live this for another PR later.

**Review by sungchun12 — APPROVED** [review]

**Pull request merged**